go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/microsoft/go-sqlcmd
 go 1.18
 
 require (
+	github.com/alecthomas/assert/v2 v2.1.0
```

Comment:
>alecthomas/assert/v2 v2.1.0 [](http://example.com/codeflow?start=12&length=27)

doesn't stretchr/testify already have asserts?

Reply:
yep, this was a mistake... removed